R/write_results.R

```diff
@@ -26,6 +26,7 @@ write_stress_test_results <- function(results, expected_loss,
 
   results_path <- file.path(get_st_data_path("ST_PROJECT_FOLDER"), "outputs")
 
+  sensitivity_analysis_vars <- paste0(sensitivity_analysis_vars, "_arg")
```

Comment:
We now have to tell the validation function to expect the bare name of the argument plus the suffix "_arg".